test/snapshot-removal/fixtures/skipped-tests/test.js

```diff
@@ -0,0 +1,22 @@
+const test = require(process.env.AVA_PATH || 'ava');
```

Comment:
Why is this needed?

Reply:
When the tests run fixtures, they first copy them to a temporary directory, so that the same fixture can be used by multiple tests concurrently. Being outside of the project's directory structure, the temporary fixtures can't `require('ava')`, so the tests instead pass AVA's location in `process.env.AVA_PATH` (see [`helpers/macros.js::withTemporaryFixture()`](https://github.com/avajs/ava/blob/52c1bd53a2a9b6aa8828708ad8c9106e9d2ee467/test/snapshot-removal/helpers/macros.js#L7)).

The `|| 'ava'` here isn't needed. It changes the error message from a missing 'AVA_PATH' (failed to require 'ava' instead of failed to require '') and it makes it less tedious to run the fixtures manually if that's useful in debugging. On the other hand, it also makes it easier to accidentally write a test case that modifies a fixture which other tests rely on being static.

I found the `|| 'ava'` mildly helpful while writing tests, but it could certainly be removed now.

Reply:
Gotcha!

Would it help to add a separate `ava.js` file in the fixtures directory and require that from the actual fixture? `AVA_PATH` is an actual private environment variable of course and there's a decent chance this becomes a pattern that is copy/pasted to other test files unnecessarily.